mantidproject/mantid
# Fix Flake8 warning associated with WorkflowAlgorithms

**IN PROGRESS**

As part of #20508, four files from `WorkflowAlgorithms` have been refactored to reduce cyclomatic complexity

**To test:**
Make sure all unit and system tests pass, and code review. In addition, if this is your area, feel free to throw some data at me to test the affected scripts with, to make sure I haven't broken anything.

Note: it was deemed acceptable to just bury the warnings associated with SANS under the rug with a `noqa` flag, as these scripts are soon to be deprecated, and a lack of unit tests makes refactoring them needlessly risky

Address part of #20508 . 

**Release Notes** 
*Does not need to be in the release notes as the change is internal*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
